github/util.py

```diff
@@ -24,8 +24,8 @@
 from github3.pulls import PullRequest
 from github3.repos.release import Release
 
-import gci.componentmodel
-import gci.componentmodel as cm
+import ocm
+import ocm as cm
```

Comment:
There is no need to import this module twice (I know this doesn't stem from this PR), but I think we shouldn't keep it that way.